packages/web-config-server/src/apiV1/dataBuilders/generic/dataDownload/rawDataValues.js

```diff
@@ -148,10 +147,6 @@ class RawDataValuesBuilder extends DataBuilder {
   sortEvents = async (events, sortKeys) => {
     if (!sortKeys) return events;
 
-    // If we are merging sorting on that key takes precedence
-    // for performance of merge and avoid ancestor lookup
-    if (sortKeys.mergeRowKey) return this.sortEventsByDataValue(events, sortKeys.mergeRowKey);
```

Comment:
We definitely need to regression test any viz that has a `mergeRowKey` and uses this data builder before releasing